<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgCtrlElk</class>
 <widget class="QDialog" name="DlgCtrlElk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>229</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Панель Управления ЭЛК</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupControl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>УПРАВЛЕНИЕ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupControlOsvPerim">
        <property name="title">
         <string>ОСВЕЩ ПЕРИМ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushOsvPerimOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushOsvPerimOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupControlCond">
        <property name="title">
         <string>ЗАПРЕТ КОНДИЦ</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushCondOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushCondOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupControlReset">
        <property name="title">
         <string>Сброс аварии</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <spacer name="spacerReset1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushReset">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>Сброс</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerReset2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupControlFire">
        <property name="title">
         <string>ПОЖАР (проверка)</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushFireOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushFireOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupControlGuard">
        <property name="title">
         <string>ОХРАНА (проверка)</string>
        </property>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="pushGuardOn">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ВКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushGuardOff">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>ОТКЛ</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QGroupBox" name="groupControlHowl">
        <property name="title">
         <string>СИРЕНА ОПУ (проверка)</string>
        </property>
        <layout class="QHBoxLayout" name="_2">
         <item>
          <spacer name="spacerHowl1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushTestHowl">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>32767</height>
            </size>
           </property>
           <property name="text">
            <string>Проверить</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacerHowl2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacerClose1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerClose2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>190</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>pushOsvPerimOn</tabstop>
  <tabstop>pushOsvPerimOff</tabstop>
  <tabstop>pushFireOn</tabstop>
  <tabstop>pushFireOff</tabstop>
  <tabstop>pushGuardOn</tabstop>
  <tabstop>pushGuardOff</tabstop>
  <tabstop>pushCondOn</tabstop>
  <tabstop>pushCondOff</tabstop>
  <tabstop>pushReset</tabstop>
  <tabstop>pushClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
